.ci/infra/terraform/README.md

```diff
@@ -57,6 +57,7 @@ Quotas that will need to be adjusted to support all tests:
 - Cloud Workstation cluster quota in `us-central1`
 - VMWare Engine nodes per region in `southamerica-west1`
 - VMWare Engine nodes across regions
+- Looker `StandardSubscriptionInstancesPerProjectPerRegion`. This must be requested manually from their team, and can't be self-served in the Cloud Console.
```

Comment:
Our CI naively stops all builds that edit `.ci/` files. Do you mind separating this out into a separate PR so we can run the test changes?

Reply:
Sure!